src/main/res/drawable/standard_button.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Newer Android versions is shaping the buttons like "oval", we used to have it like square with slightly rounded corders. 
No way to shape buttons with rounded corners as we do in CSS. Here we create a drawable and use it as button's background, this is supported in Android 5+. 
So to keep consistency between Android versions and as it was originally.